packages/datetime/src/dateRangePicker.tsx

```diff
@@ -225,11 +227,11 @@ export class DateRangePicker extends AbstractPureComponent<IDateRangePickerProps
         const { leftView, rightView } = this.state;
         const disabledDays = this.getDisabledDaysModifier();
 
-        const dayPickerBaseProps: ReactDayPicker.Props = {
-            enableOutsideDays: true,
+        const dayPickerBaseProps: DayPickerProps = {
             locale,
             localeUtils,
             modifiers,
+            showOutsideDays: true,
```

Comment:
Curious if this prop should be configurable given it's always hidden since this PR https://github.com/palantir/blueprint/pull/586/files#r98813760?

Reply:
We can leave this as is for now. Seems like changing it here would be out of scope for this PR.